welcome/index.adoc

```diff
@@ -283,9 +283,9 @@ Learn about how to deploy and use a cluster to aggregate logs for a range of {pr
 services. Cluster logging collects, stores, and visualizes logging data from hosts and
 applications, whether coming from multiple containers or even deleted pods.
 
-- **xref:../monitoring/cluster_monitoring/about-cluster-monitoring.adoc#about-cluster-monitoring[Monitor clusters]**: Learn to
-xref:../monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack].
-Once your monitoring is configured, use the Web UI to xref:../monitoring/cluster_monitoring/prometheus-alertmanager-and-grafana.adoc#prometheus-alertmanager-and-grafana[access Prometheus, Alertmanager, and Grafana].
+- **xref:../monitoring/understanding-the-monitoring-stack.adoc#understanding-the-monitoring-stack[Monitor clusters]**: Learn to
+xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack].
+Once your monitoring is configured, use the Web UI to xref:../monitoring/accessing-monitoring-dashboards.adoc[monitoring dashboards].
```

Comment:
Looks like the build is currently failing because of this path. It's missing an anchor ID, but at first glance I don't see an accessing-monitoring-dashboards.adoc file.

Reply:
@ahardin-rh thank you for your message. I have a lot of cross-references to fix because I have restructured the whole book. The restructuring is in line with requirements from our Monitoring team. I have almost finished adding new content into this PR and will fix all cross-references before I request further peer, QE and SME review. Thanks again!

Reply:
@pneedle-rh I see. I'm here to help if you need anything!

Reply:
Thank you!